core/eolearn/core/core_tasks.py

```diff
@@ -90,7 +90,9 @@ def __init__(
         features: FeaturesSpecification = ...,
         overwrite_permission: OverwritePermission = OverwritePermission.ADD_ONLY,
         compress_level: int = 0,
+        *,
         use_zarr: bool = False,
+        temporal_selection: None | slice | list[int] = None,
```

Comment:
Not sure if I noticed this before or not, but now there seems to be some redundancy in parameters, right?

If `temporal_selection` is not `None`, then usage of `zarr` is/should be implied. The issue is the init save, I guess. 

Did you have any thought on this already?

Reply:
yeah, you can save in zarr format and not specify anything. If you have a zarr based eo-grow project probably only a few pipelines will work with temporal selection